spec/features/visitors/navigation_spec.rb

```diff
@@ -1,18 +1,13 @@
-# Feature: Navigation links
-#   As a visitor
-#   I want to see navigation links
-#   So I can find home, sign in, or sign up
-feature 'Navigation links', :devise do
-
-  # Scenario: View navigation links
-  #   Given I am a visitor
-  #   When I visit the home page
-  #   Then I see "home," "sign in," and "sign up"
+feature 'Navigation links', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.